manifest-generation/config-from-cf-internal.yml

```diff
@@ -69,6 +74,8 @@ properties:
     server_key: (( merge ))
   loggregator:
     etcd:
+      ca_cert: (( merge || nil ))
```

Comment:
I know it's repeated elsewhere, but `(( merge || nil ))` can just be `~` or `null` (not `nil`, though).